EligibilityCheckActivity/handler.ts

```diff
@@ -73,6 +76,24 @@ export const getEligibilityCheckActivityHandler = (
           })
           .map(_ => ({ dsu: _, fiscalCode }));
       })
+      .chain(data => {
+        if (
+          !data.dsu.DatiIndicatore?.Componenti?.some(
+            familyMember => familyMember.CodiceFiscale === data.fiscalCode
+          )
+        ) {
+          return taskEither.of({
+            dsu: {
+              ...data.dsu,
+              DescrizioneErrore:
+                "Missing requester fiscal code inside family members array",
+              Esito: EsitoEnum.ERRORE_INTERNO
+            },
+            fiscalCode: data.fiscalCode
+          });
+        }
```

Comment:
As we are basically augmenting the abilities of `ConsultazioneSogliaIndicatore` service, does it make sense to move such logic inside `ConsultazioneSogliaIndicatore` itself, and keep the activity logic neat?

Reply:
Yes, it's could be done.

Reply:
fixed in 233cd6a